src/Navbar.tsx

```diff
@@ -192,15 +197,20 @@ class Navbar extends React.Component<any, State> {
                   >
                     SIGN&nbsp;IN
                   </Button>
-                  <Modal 
+                  <Modal
                     animation={false}
-                    backdrop='static'
+                    backdrop="static"
                     keyboard={false}
-                    onHide={handleCloseLoginDialog} 
+                    onHide={() => {
+                      handleCloseLoginDialog()
+                    }}
```

Comment:
Fixes a linting error where the type of handleCloseLoginDialog didn't match the permitted type of onHide